finished layout cards

kapesha opened the pull request:
- [DEMO LINK](https://kapesha.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://kapesha.github.io/layout_product-cards/report/html_report/)

lerastarynets reviewed (approved):
You're doing brilliantly, keep it up!

lerastarynets reviewed (approved):
You're doing brilliantly, keep it up!

lerastarynets reviewed (approved):
Did you forget to redeploy the project?) Previous comments aren’t fixed

kapesha commented:
I had some problems with correct deploying of project, so I just deleted old one and made all from the start, no links must work correctly. Comments from previous repo i fixed, as i remember
